applications/nsmrs/go.mod

```diff
@@ -14,6 +14,8 @@ require (
 )
 
 replace (
+	github.com/networkservicemesh/networkservicemesh => ../../
+	github.com/networkservicemesh/networkservicemesh/applications/nsmrs => ./
```

Comment:
Why we need to do replace `github.com/networkservicemesh/networkservicemesh/applications/nsmrs => ./ ` inside module `module github.com/networkservicemesh/networkservicemesh/applications/nsmrs`  ?

Reply:
I did have some issues wit **go get** of that module and that replace helped.

Reply:
it is interesting. 
Should we add a check on CI to check that our modules are `go gettable`?